src/i18n/en/common.json

```diff
@@ -113,7 +113,8 @@
       "loginButton": "Log in",
       "preview": {
         "heading": "Area search application preview",
```

Comment:
Title change has been done to Finnish. Should we remove translation of the heading or at least translate it ourself?

Reply:
Good point, it should be in the English translations too. I will add it.

Reply:
Realized that the ordering of numbers was directly taken from abstract, but it has one extra step as 1, so it was off by one. Fixed it to 3.